uptane/encoding/asn1_definitions.py

```diff
@@ -589,7 +589,7 @@ class VehicleVersionManifestSigned(univ.Sequence):
     namedtype.NamedType('vehicleIdentifier', Identifier().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 0))),
     namedtype.NamedType('primaryIdentifier', Identifier().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 1))),
     namedtype.NamedType('numberOfECUVersionManifests', Length().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 2))),
-    namedtype.NamedType('ecuVersionManifests', ECUVersionManifests().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 3))),
+    namedtype.NamedType('ecuVersionManifests', ECUVersionManifests().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 3))), # Should this be tagFormatConstructed?
```

Comment:
Formally, tag form ("primitive" or "constructed") is tied to the base ASN.1 type (which can be "primitive" of "constructed"). 

Primitive (scalar) types are generally encoded with a "primitive" tag, however they may sometimes be encoded in "constructed" form. Cases of primitive-turned-constructed encoding include "explicit tagging" and indefinite-length encoding situations.

I do not think, however, that it is valid to encode originally constructed types (such as SEQUENCE OF) with a "primitive" tag.

With pyasn1 implementation, "implicit" tagging implies [inheriting](https://github.com/etingof/pyasn1/blob/master/pyasn1/type/tag.py#L312) parent tag form. That hopefully explains why DER does not change when you flip tag form.

Therefore I think it would make sense to use [tagFormatConstructed](https://github.com/awwad/uptane/blob/814eaf5e99b1acce901f9acd7df483fe9bb5fff0/uptane/encoding/asn1_definitions.py#L592) even though it does not presently influence the encoding.